src/internal/forms.py

```diff
@@ -176,3 +178,35 @@ class Meta:
         widgets = {
             'date': SemanticDateInput(),
         }
+
+
+class LoreForm(forms.ModelForm):
+    class Meta:
+        model = Lore
+        fields = ('title', 'content')
+        widgets = {
+            'text': CKEditorWidget()
+        }
+
+    def clean(self):
+        cleaned_data = self.cleaned_data
+        title = cleaned_data.get('title')
+        title = title.replace('ø', 'o')
+        title = title.replace('æ', 'ae')
+        slug = slugify(title)
```

Comment:
I agree, much better solution. The only reason I did that very ad-hoc replace-solution in the first place was because I didn't know it was possible to fix this by simply writing `allow_unicode=True`